### ISSUE TITLE
How about the performance of multi-class detection ?

### ISSUE THREAD

**Issue by catsdogone** [xiaohaoChen/rrc_detection#15]
@xiaohaoChen @jimmy-ren 

Hi! Thanks for your code and paper.

It seems that you code is designed for one class training. 

If I want to train a multi-class model, a model that can detect car, person and bicycle at the same time, Should I change the loss function? And do you think the multi-class model will has good performance as described in your paper?

Thank you very much.

**Comment by jimmy-ren** [comment]
What we found during the experiments on KITTi was using a single model for car and a single model for both pedestrian and cyclist gave the best results. Many other papers in the literature seem to did the same. This may due to the limitation (limited groundtruth) of the KITTI dataset. In theory, if you have enough data, train a model to detect all the categories shall also give good results.

**Comment by joonjeon** [comment]
Hi, there!

Could you take a look at my issue?:
https://github.com/xiaohaoChen/rrc_detection/issues/13

Thanks in advance.


2017-06-19 17:07 GMT+09:00 Jimmy Ren <notifications@github.com>:

**Comment by zhanghanduo** [comment]
@jimmy-ren @xiaohaoChen 
Is it possible to provide the trained model weights for pedestrian and cyclist classes ?
Thank you

**Comment by GreyCoder25** [comment]
@jimmy-ren @xiaohaoChen
Is it possible to provide the trained model weights for pedestrian and cyclist classes ?
Thank you
